extensions/amp-script/0.1/test/validator-amp-script.out

```diff
@@ -102,44 +102,47 @@ amp-script/0.1/test/validator-amp-script.html:82:2 The relative URL '/foo.js' fo
 |    <amp-script layout=container src="http://not.https.url"></amp-script>
 >>   ^~~~~~~~~
 amp-script/0.1/test/validator-amp-script.html:85:2 Invalid URL protocol 'http:' for attribute 'src' in tag 'amp-script'. (see https://amp.dev/documentation/components/amp-script)
-|
+|    
 |    <!-- Invalid: 'development' attribute. -->
 |    <amp-script layout=container src="https://example.com/foo.js" development></amp-script>
 >>   ^~~~~~~~~
-amp-script/0.1/test/validator-amp-script.html:88:2 The attribute 'development' may not appear in tag 'amp-script'. (see https://amp.dev/documentation/components/amp-script)
+amp-script/0.1/test/validator-amp-script.html:88:2 The attribute 'development' in tag 'amp-script' is deprecated - use 'the data-ampdevmode attribute on the root html element' instead. (see https://github.com/ampproject/amphtml/issues/24987)
+|
+|    <!-- Invalid: 'data-ampdevmode' attribute. -->
```

Comment:
this states the result should be invalid but there isn't a corresponding validator error for it. perhaps this .out file needs to be updated with `gulp validator -- update-test`.

Reply:
Thanks for catching this! After changing tack with how I was implementing this fix I forgot to update both the docs and the validator. Should be fixed now.

This PR as a whole is blocked on getting consensus around https://github.com/ampproject/amphtml/issues/24987#issuecomment-550490860.